tests/bin/libc.h

```diff
@@ -73,3 +73,18 @@ int is_enarx() {
         default: return 0;
     }
 }
+
+ssize_t get_att(void *nonce, size_t nonce_len, void *buf, size_t buf_len, size_t *technology) {
+    ssize_t rax;
+    ssize_t tech;
+    register size_t r10 __asm__("r10") = buf_len;
+
+    asm(
+        "syscall"
```

Comment:
Indent.

Reply:
@npmccallum How do you tend to check that your indentation is correct (here and in general)? I tend to mess up on this.

Reply:
Basically, you just need to configure your editor for automatic indenting. So it depends on your editor.

Reply:
Technically, `"syscall"` is the correct indent. It is the lines below that are incorrect.